ui/plugin-system/src/runtime/index.ts

```diff
@@ -12,8 +12,9 @@
 // limitations under the License.
 
```

Comment:
hmm maybe exports need to be adjusted? Seeing `import { OptionsEditorProps, usePlugin } from '@perses-dev/plugin-system';` error in CI

Reply:
Probably some circular imports going on. You probably need to adjust the imports in `PluginSpecEditor` (the stuff it's importing from `@perses-dev/plugin-system`) to be local imports now that it's in the same package as that stuff.

Reply:
Ah yeah, let me see if that fixes it

Reply:
Ah yeah I’ve done this before too, we should install that package we use internally, I think: https://www.npmjs.com/package/circular-dependency-plugin